packages/modelviewer.dev/data/docs.json

```diff
@@ -7,55 +7,55 @@
         "htmlName": "src",
         "description": "<p>The URL to the 3D model. This parameter is required for <span class=\"attribute\">&lt;model-viewer&gt;</span> to display. Only <a href=\"https://github.com/KhronosGroup/glTF/tree/master/specification/2.0\">glTF</a>/<a href=\"https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#glb-file-format-specification\">GLB</a> models are supported, see <a href=\"https://github.com/google/model-viewer#supported-formats\">Supported Formats</a>.</p>",
         "default": [
-          "empty string",
           "String",
-          "any legal filepath"
+          "any legal filepath",
+          "empty string"
```

Comment:
I like the sticky headings like "Loading", but could we add the current category? Like "Loading: Attributes"